.env.example

```
@@ -25,6 +25,12 @@ RABBITMQ_TLS=false
 RABBITMQ_EXCHANGE=contact_messages
 RABBITMQ_QUEUE=contact_messages
 RABBITMQ_RETRY_DELAYS=1m,5m,30m,2h,12h
+RABBITMQ_HEARTBEAT=10s
+RABBITMQ_PUBLISHER_CONFIRMS=false
+RABBITMQ_RECONNECT=true
+RABBITMQ_RECONNECT_MAX_ATTEMPTS=0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_RECONNECT_MAX_ATTEMPTS key should go before the RABBITMQ_RETRY_DELAYS key

(UnorderedKey)